pkg/cli/config.go

```diff
@@ -53,16 +53,33 @@ type HTTPBasicAuthnConfig struct {
 }
 
 type Config struct {
+	// Default service, service to be used in case none other was selected.
+	SystemDefaultService string `json:"system_default_service,omitempty"`
+	// [OPTIONAL] If set, it overrides the `SystemDefaultService` parameter.
+	UserDefaultService   string              `json:"user_default_service,omitempty"`
+	Services             map[string]*Service `json:"services,omitempty"`
+	ConnectionControlDir string              `json:"connection_control_dir,omitempty"`
+	KeepLogFilesDays     int                 `json:"keep_log_files_days,omitempty"`
+}
+
+type Service struct {
 	ServiceURL                string       `json:"service_url,omitempty"`
 	Zone                      string       `json:"zone,omitempty"`
 	HTTPProxy                 string       `json:"http_proxy,omitempty"`
-	ConnectionControlDir      string       `json:"connection_control_dir,omitempty"`
-	KeepLogFilesDays          int          `json:"keep_log_files_days,omitempty"`
 	BuildAPICredentialsSource string       `json:"build_api_credentials_source,omitempty"`
-	Host                      HostConfig   `json:"host,omitempty"`
+	Host                      *HostConfig  `json:"host,omitempty"`
 	Authn                     *AuthnConfig `json:"authn,omitempty"`
 }
 
+func (c *Config) Service() *Service {
```

Comment:
This looks like it should be named `DefaultService`.

Reply:
yes indeed. done.